### PR TITLE
Name regex

### LINKED ISSUE #92: Upload new dockerised ETL
## Task Description

- Upload new docker version that has the brand new scripts ready to be pushed for the ECR

## User Stories

- As a developer its important that we always dockerise our programs into containers so that we can access them via AWS services

## Relevant Files [If Available] 

-

### LINKED ISSUE #179: Fix judge_name regex
## Task Description

- Currently name regex is too broad and is picking up extra pieces that it shouldn't

## User Stories

- As a developer, I need to make sure my regex is as precise as possible, so that I can be sure I am picking up only relevant data.

## Relevant Files [If Available] 

- `extract.py`

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
Fixed some of the regex, less cases are being dropped now.

Closes #179 
Closes #92

### CODE REVIEW

**Comment by luciegamby-sigmalabs** [comment]
Pushed the ETL image to the repo

File: pipeline/extract.py [hunk 1]
```diff
@@ -162,22 +162,13 @@ def parse_pdf(court_case: dict):
...
-        if not judge:
-            judge = re.search(r"(?<=Before:\n)([A-Z a-z]*)", first_page)
-        if not judge:
-            judge = re.search(r"(?<=BEFORE:\n)([A-Z a-z]*)", first_page)
-        if not judge:
-            judge = re.search(r"(THE HONOURABLE [A-Z a-z]*)", first_page)
-        court_case["judge_name"] = judge.group(1).strip()
+        judge = re.search(
+            r"(?:before {0,}: {0,}\n{0,1} {0,1}\n{0,}|the honourable )([a-z .]*)", first_page.lower())
+        court_case["judge_name"] = judge.group(1).strip().replace('.', '')
```
Comment (MahinRahman8901) [inline pipeline/extract.py:167]: I like how you compressed everything into three lines

File: pipeline/extract.py [hunk 2]
```diff
@@ -187,7 +178,8 @@ def parse_pdf(court_case: dict):
         if not court_date:
             court_date = re.search(
                 r"([\w]{0,},? ?[0-9]{1,2} [A-Z|a-z]+ [0-9]{2,4})|([0-9]{2}[/][0-9]{2}[/][0-9]{2,4})", first_page)
-        court_case["date"] = court_date.group(1).strip()
+        court_case["date"] = court_date.group(1).strip().replace(
+            "1st", '1').replace("nd", "").replace("rd", "").replace("th", "")
```
Comment (MahinRahman8901) [inline pipeline/extract.py:182]: Good way to replace the data titles

**Review by MahinRahman8901 — APPROVED** [review]

File: pipeline/extract.py (hunk 1 above)
Comment (luciegamby-sigmalabs) [reply pipeline/extract.py:167]: 😅

**Pull request merged**